oozie/src/main/java/org/apache/falcon/oozie/feed/FSReplicationWorkflowBuilder.java

```diff
@@ -58,20 +58,25 @@ public FSReplicationWorkflowBuilder(Feed entity) {
         addHDFSServersConfig(replication, src, target);
         addAdditionalReplicationProperties(replication);
         enableCounters(replication);
-        addTransition(replication, SUCCESS_POSTPROCESS_ACTION_NAME, FAIL_POSTPROCESS_ACTION_NAME);
-        workflow.getDecisionOrForkOrJoin().add(replication);
 
-        //Add post-processing actions
-        ACTION success = getSuccessPostProcessAction();
-        addHDFSServersConfig(success, src, target);
-        addTransition(success, OK_ACTION_NAME, FAIL_ACTION_NAME);
-        workflow.getDecisionOrForkOrJoin().add(success);
+        if (!Boolean.parseBoolean(ENABLE_POSTPROCESSING)){
```

Comment:
Can we add a method in the super class and use it across FSReplicationWorkflowBuilder, FeedRetentionWorkflowBuilder and ProcessExecutionWorkflowBuilder?